svgdigitizer/__main__.py

```diff
@@ -168,8 +173,13 @@ def digitize(svg, sampling_interval, outdir):
     default=None,
     help="write output files to this directory",
 )
+@click.option('--algorithm',
```

Comment:
We should probably not duplicate these arguments, i.e., instead write:

```py
skewed = click.option(...)
```

and then
```patch
-@click.option(...)
+@skewed
```